project_name/images/models.py

```diff
@@ -0,0 +1,31 @@
+from django.db import models
+from django.db.models.signals import pre_delete
+from django.dispatch.dispatcher import receiver
+
+from wagtail.wagtailimages.models import AbstractImage, AbstractRendition
+
+
+class Image(AbstractImage):
+    pass
+
+
+class Rendition(AbstractRendition):
```

Comment:
I've no understanding of what need you're addressing by creating a custom Rendition here, or why we need to define (or is it redefine?) custom receivers, below. Given this is a template, a line or two of comment would be particularly useful.